frontend/src/modules/editor/components/KeyboardShortcuts.js

```diff
@@ -0,0 +1,210 @@
+/* @flow */
+
+import * as React from 'react';
+import onClickOutside from 'react-onclickoutside';
+import { Localized } from 'fluent-react';
+
+import './KeyboardShortcuts.css';
+
+
+type Props = null;
+
+
+type State = {|
+    visible: boolean,
+|};
+
+
+/*
+ * Shows a list of keyboard shortcuts.
+ */
+export class EditorSettingsBase extends React.Component<Props, State> {
+    constructor() {
+        super();
+        this.state = {
+            visible: false,
+        };
+    }
+
+    toggleVisibility = () => {
+        this.setState((state) => {
+            return { visible: !state.visible };
+        });
+    }
+
+    // This method is called by the Higher-Order Component `onClickOutside`
+    // when a user clicks outside the keyboard shortcut overlay.
+    handleClickOutside = () => {
+        this.setState({
+            visible: false,
+        });
+    }
+
+    render() {
+        return <div className="keyboard-shortcuts">
+            <Localized
+                id="editor-keyboard-shortcuts-button"
+                attrs={{ title: true }}
+            >
+                <div
+                    className="selector far fa-keyboard"
+                    title="Keyboard Shortcuts"
+                    onClick={ this.toggleVisibility }
+                />
+            </Localized>
+
+            { !this.state.visible ? null :
+            <div
+                className="overlay"
+                onClick={ this.toggleVisibility }
+            >
+                <Localized id="editor-keyboard-shortcuts-overlay-title">
+                    <h2>Keyboard Shortcuts</h2>
+                </Localized>
+                <table>
+                    <tbody>
+                        <tr>
+                            <Localized id="editor-keyboard-shortcuts-save-translation">
+                                <td>Save Translation</td>
+                            </Localized>
+                            <td>
+                                <Localized id="editor-keyboard-shortcuts-save-translation-shortcut">
+                                    <span>Enter</span>
```

Comment:
These `<span>` should be in the string here too, I guess, otherwise they won't be part of the rendered content and that might break the styling?